docs/UPSTREAMING.md

```diff
@@ -108,7 +106,7 @@ Further, the boot process is a bit "backwards", at least for the live `rootfs.im
 
 * `sgdisk` - with specific patches listed [here](../pkg/gpt-tools/patches)
 * `cgpt` - works with ChromeOS-specific GPT partitioning
-* [zboot](../pkg/gpt-tools/files/zboot) - a script whose purpose is unknown. It is not referenced elsewhere that we can find.
+* [zboot](../pkg/gpt-tools/files/zboot) - a script that is the main entry point for EVE Go code querying and manipulating the state of partitions
```

Comment:
What calls it and when? Is it some of the go-provision code? Or something else?

Reply:
go-provision calls zboot.

Reply:
Oh I see it now. So if `go-provision` wants to find out which partition is currently active (a or b), or switch it, etc., it uses `zboot` which is simply a convenience wrapper?